app/src/main/java/com/conupods/OutdoorMaps/Models/PointsOfInterest/Photo.java

```diff
@@ -42,7 +42,7 @@ public void setHeight (String height)
     @Override
     public String toString()
     {
-        return "ClassPojo [photo_reference = "+photo_reference+", width = "+width+", html_attributions = "+", height = "+height+"]";
+        return "ClassPojo [photo_reference = "+ photoReference +", width = "+width+", html_attributions = "+", height = "+height+"]";
```

Comment:
Do not change the name of this attribute, it must match the attribute name that is within the JSON response object. Revert change.